source/solutions-library/retail-as-you-type-suggestions.txt

```diff
@@ -216,34 +202,32 @@ The ``type`` field is indexed as both a ``token`` field, for ``equals`` or ``in`
 filtering, and a ``stringFacet`` field to provide a means to get counts
```

Comment:
above (can't comment) the sentence "a multitude ("multi") of ways (the feature is called
“multi”-analyzers)." can be rewritten as something less casual